pyproject.toml

```diff
@@ -21,7 +21,7 @@ python = ">=3.9, <4"
 requests = "^2.31.0"
 tqdm = "^4.66.0"
 geojson = "3.1.0"
-geopandas = "^0.13.2"
+geopandas = "^0.14.0"
```

Comment:
Why can't we set in generic form to anything with this major version?

Reply:
setted to >=0.0.0, <1.0.0 https://python-poetry.org/docs/dependency-specification/